docs/guides/integration/github.md

```diff
@@ -229,6 +229,12 @@ steps:
     run: uv run pytest tests
 ```
 
+!!! tip
+
+    The
+    [`UV_PROJECT_ENVIRONMENT` setting](../../concepts/projects.md#custom-project-environment-paths) can
+    be used to install to the system Python environment instead of creating a virtual environment.
```

Comment:
For some reason the syntax highlighting here is different (dark rather than light blue) so maybe just sanity check that it's being rendered correctly.

Reply:
👍 looks good rendered

<img width="836" alt="Screenshot 2024-09-03 at 6 43 17 PM" src="https://github.com/user-attachments/assets/f757e945-960a-4393-bedc-dc759b557031">